.github/workflows/release.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build and Push Database Exporter
+
+on:
+  push:
+    tags:
+      - "*.*.*-*"
+
+env:
+  UBUNTU_VERSION: '22.04'
+  RUST_IMAGE_TAG: '1.73.0'
+  S3_BUCKET: s3://eur2ccd.concordium.com
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  release-euro2ccd:
+    runs-on: ubuntu-latest
+    environment: release
```

Comment:
Have you ensured that this can only apply on the tag with above pattern?

Furthermore we want a +1 from another person than yourself on the action as well